ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/http/JsonRpcHandler.java

```diff
@@ -48,6 +48,7 @@ public void handle(final RoutingContext context) {
       LOG.trace("Request body = {}", context.getBodyAsString());
       final JsonRpcRequest request =
           jsonDecoder.decodeValue(context.getBody(), JsonRpcRequest.class);
+      context.put("JsonRpcId", request.getId());
```

Comment:
Perhaps create this as constant so this can be also used in the JsonRpcErrorHandler as the value to pull out of the context

Reply:
done